pkgs/applications/audio/monophony/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, fetchFromGitLab
+, python3Packages
+, wrapGAppsHook4
+, gst_all_1
+, gobject-introspection
+, gtk4
+, yt-dlp
+, libadwaita
+, libsoup_3
+, glib-networking
+,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "monophony";
+  version = "2.2.2";
+  format = "other";
+
+  sourceRoot = "source/source";
+  src = fetchFromGitLab {
+    owner = "zehkira";
+    repo = "monophony";
+    rev = "v${version}";
+    hash = "sha256-vAX9ac706fKVYW1jjgUJ2GvGRWeytYfgf7Lal+gEm04=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    brotli
+    gst-python
+    mutagen
+    ordered-set
+    pycryptodomex
+    pygobject3
+    requests
+    websockets
+    ytmusicapi
+    mpris-server
+  ];
+
+  nativeBuildInputs = [
+    gobject-introspection
+    wrapGAppsHook4
+    python3Packages.nuitka
+  ];
+
+  buildInputs =
+    [
+      libadwaita
+      libsoup_3
+      glib-networking
+    ]
+    ++ (with gst_all_1; [
+      gst-plugins-base
+      gst-plugins-good
+      gstreamer
+    ]);
+
+  makeFlags = [ "prefix=$(out)" ];
+
+  preFixup = ''
+        gappsWrapperArgs+=(
+    	  --prefix PYTHONPATH : "$PYTHONPATH"
+    	  --prefix LD_LIBRARY_PATH : "${libsoup_3.out}/lib"
+    	  --prefix PATH : "${lib.makeBinPath [yt-dlp]}"
+        )
+  '';
+
+  meta = with lib; {
+    homepage = "https://gitlab.com/zehkira/monophony";
+    description = "Monophony is a free and open source Linux app for streaming music from YouTube. It has no ads and does not require an account.";
```

Comment:
this seems more like a longDescription. description should be short, not mention the pkg name, nor end in a period.

Reply:
Oh, thanks!

Reply:
Should be fixed.